modules/nf-core/ltrharvest/tests/main.nf.test

```diff
@@ -26,7 +26,31 @@ nextflow_process {
                 { assert process.success },
```

Comment:
You can combine the snapshots like so:
```
                { assert process.success },
                { assert snapshot(process.out.gff3,
                                               process.out.versions).match() },
                { assert path(process.out.scn[0][1]).text.contains("46510803 46520182 9380 46510803 46510940 138 46520042 46520182 141 86.52 0 chr21") },
```

Reply:
Thank you! This is a very useful comment. I have used it to simplify the snapshotting logic!